packages/@react-aria/table/src/useTableColumnResize.ts

```diff
@@ -128,18 +149,28 @@ export function useTableColumnResize<T>(props: AriaTableColumnResizeProps<T>, st
     onMoveEnd(e) {
       let {pointerType} = e;
       columnResizeWidthRef.current = 0;
-      if (pointerType === 'mouse') {
+      if (pointerType === 'mouse' || (pointerType === 'touch' && !resizeOnFocus)) {
         endResize(item);
       }
     }
   });
 
+  let onKeyDown = useCallback((e) => {
+    if (editModeEnabled) {
+      moveProps.onKeyDown(e);
+    }
+  }, [editModeEnabled, moveProps]);
+
+
   let min = Math.floor(state.getColumnMinWidth(item.key));
   let max = Math.floor(state.getColumnMaxWidth(item.key));
   if (max === Infinity) {
     max = Number.MAX_SAFE_INTEGER;
   }
   let value = Math.floor(state.getColumnWidth(item.key));
+  let modality = useInteractionModality();
+  let description = !resizeOnFocus && modality === 'keyboard' && !isResizing.current ? stringFormatter.format('resizerDescription') : undefined;
+  let descriptionProps = useDescription(description);
```

Comment:
I had initially included a description to let the user know that Enter/Escape/Space/Tab would exit resize mode, but testing in screen readers showed that the update to the description as you entered resizing wouldn't be announced. I could do something with a live region announcer perhaps instead but I figured that second announcement is extraneous seeing as the input is announced as a slider and users would likely hit tab to blur away from the slider hence exiting resize mode properly